WebDriverAgentLib/Commands/FBDebugCommands.m

```diff
@@ -22,6 +23,9 @@ + (NSArray *)routes
 {
   return
   @[
+    [[FBRoute GET:@"/source/:type"] respondWithTarget:self action:@selector(handleGetSourceCommand:)],
+    [[FBRoute GET:@"/source/:type"].withoutSession respondWithTarget:self action:@selector(handleGetSourceCommand:)],
+    // TODO: Kill source methods, that support json representation only
```

Comment:
this should be another PR. Also, we need to sync with Appium, since they use this endpoint as well. Whet we can do for now - use the single _ handleGetSourceCommand_ method for all endpoints to avoid code duplication